jumpscale/sals/marketplace/solutions.py

```diff
@@ -45,6 +45,9 @@ def list_gitea_solutions(self, username, next_action=NextAction.DEPLOY, sync=Tru
     def list_mattermost_solutions(self, username, next_action=NextAction.DEPLOY, sync=True):
         return self._list_proxied_solution("mattermost", next_action, sync, "nginx", owner=username)
 
+    def list_meetings_solutions(self, username, next_action=NextAction.DEPLOY, sync=True):
+        return self._list_proxied_solution("meetings", next_action, sync, owner=username)
```

Comment:
We should specify the listing type as the default is trc, in the solution it's nginx